spec/std/http/client/response_spec.cr

```diff
@@ -1,7 +1,7 @@
 require "spec"
 require "http/client/response"
 
-class HTTP::Client
+class HTTP::Client::Reponse
```

Comment:
Why?

Reply:
When the spec is run in isolation, `http/client.cr` is not included and thus `HTTP::Client` is a module, not a class. For the response spec it is completely irrelevant what kind of type `HTTP::Client` is.

Reply:
@straight-shoota anyhow there's a typo: `Reponse` -> `Response`